source/rafcon/gui/mygaphas/items/state.py

```diff
@@ -59,13 +59,16 @@ class StateView(Element):
 
     _map_handles_port_v = {}
 
-    def __init__(self, state_m, size, hierarchy_level):
+    def __init__(self, state_m, size, custom_background_color, background_color, hierarchy_level):
         super(StateView, self).__init__(size[0], size[1])
         assert isinstance(state_m, AbstractStateModel)
         # Reapply size, as Gaphas sets default minimum size to 1, which is too large for highly nested states
         self.min_width = self.min_height = 0
         self.width = size[0]
         self.height = size[1]
+        self.custom_background_color = custom_background_color
```

Comment:
Same here. Set `[custom_]background_color` to `False` if there is no custom color, otherwise store the color value.